### PR TITLE
contrib/aria2: new package

### PR DESCRIPTION
**Pull request by nullobsi**
I believe aria2 has a library that could be built too, but I haven't figured out how to achieve this with a subpackage yet or if it should be -devel or -libs.

### CODE REVIEW

**Comment by nekopsykose** [comment]
just leave the lib out for now

**Comment by ttyyls** [comment]
try this for an update.py
```
url = "https://github.com/aria2/aria2/tags"
pkgname = "release"
```

File: contrib/aria2/template.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+]
+makedepends = [
+    "openssl-devel",
+    "zlib-devel",
+    "libxml2-devel",
+    "c-ares-devel",
+    "libuv-devel",
+    "libssh2-devel",
+    "cppunit-devel",
+    "sqlite-devel",
```
Comment (nekopsykose) [inline contrib/aria2/template.py:47]: sort these

File: contrib/aria2/template.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+]
+pkgdesc = "Lightweight multi-protocol download utility"
```
Comment (nekopsykose) [inline contrib/aria2/template.py:49]:
```suggestion
pkgdesc = "Multi-protocol download utility"
```

**Comment by nekopsykose** [comment]
seems to work with hardening = vis, cfi too

**Pull request merged**